src/controllers/index.controller.js

```diff
@@ -153,6 +154,7 @@ async function start(req, res) {
         seed,
         multisigData,
         passphrase: req.body.passphrase,
+        allowPassphrase: config.allowPassphrase,
```

Comment:
This is because instead of importing the settings module on the helper, I think it should receive the flag as a parameter